nunaliit2-couch-application/src/main/atlas_couchapp/_attachments/tools/css/metadata.css

```diff
@@ -0,0 +1,3 @@
+.metadataForm {
+    padding: 5px;
```

Comment:
Some files are using tabs, others spaces. For consistency with other Nunaliit files, they probably should all be tabbed. 

If you want, you can re-tab a file quickly in vi using `:%retab!` based on your tab settings (converts the number of spaces = to a tab to a tab character). However i'm sure there are other programs that can do it as well.

Reply:
The IDE's have a mind of their own sometimes.  They usually try to match what's in the file, but some of them will apply defaults so it ends up mixing.  I now have IntelliJ set to tabs (defaulted to spaces).  I'm not longer using VSCode for JS editing, so that shouldn't be a problem.  I don't want to go back and fix these files, but the IDE settings should prevent further issues.